config/application.rb

```diff
@@ -30,5 +30,12 @@ class Application < Rails::Application
     config.action_view.field_error_proc = Proc.new do |html_tag, instance|
       "<span class='field_with_errors'>#{html_tag}</span>".html_safe
     end
+
+    host = Rails.application.secrets.host
+    if Rails.env.production? && host
+      config.action_mailer.default_url_options = {
+        host: host, protocol: 'https://'
```

Comment:
This will not work on demo, as we don't have https working there for RG app.
Maybe it would be better to put protocol to the config as well?